packages/base/room.gts

```diff
@@ -382,56 +388,62 @@ export class RoomCard extends Card {
       let index = cache.size;
       let newMessages = new Map<string, MessageCard>();
       for (let event of this.events) {
-        if (event.type !== 'm.room.message') {
-          continue;
-        }
-        let event_id = event.event_id;
-        let update = false;
-        if (event.content['m.relates_to']?.rel_type === 'm.replace') {
-          event_id = event.content['m.relates_to'].event_id;
-          update = true;
-        }
-        if (cache.has(event_id) && !update) {
-          continue;
-        }
+        if (this.roomId == event.room_id) {
```

Comment:
The room base card is somehow indexing events not associated with this room. I decided to put the guard here